terms-and-conditions-page/template.html

```diff
@@ -0,0 +1,80 @@
+{% extends "look-and-feel/layouts/two_thirds.html" %}
+{% from "look-and-feel/components/phase_banner.njk" import phaseBanner %}
+
+{% block head -%}
+    <link href="{{ asset_path }}main.css" media="screen" rel="stylesheet" />
+{% endblock %}
+
+{% set phase="ALPHA" %}
```

Comment:
If you rebase master then you can remove here until line 14 and the beta banner should still appear

Reply:
okey dokey. DONE